Applications/spire/source/ui/window.cpp

```diff
@@ -141,29 +145,28 @@ void window::set_border_stylesheet(const QColor& color) {
 }
 
 void window::update_resize_boxes() {
-  auto padding_size = scale(20, 20);
   auto top_left = mapToGlobal(geometry().topLeft());
   if(!m_resize_boxes.is_initialized()) {
     m_resize_boxes.emplace();
   }
-  m_resize_boxes->m_top_left = QRect(top_left.x() - padding_size.width(),
-    top_left.y() - padding_size.height(), padding_size.width(),
-    padding_size.height());
+  m_resize_boxes->m_top_left = QRect(top_left.x() - PADDING_SIZE,
+    top_left.y() - PADDING_SIZE, PADDING_SIZE,
+    PADDING_SIZE);
```

Comment:
Fix 154 onto 153 if possible. Same goes for all the other lines. Try to fit in as much into the 80 columns.